firefox-ios/Client/Frontend/PasswordGenerator/View/PasswordGeneratorHeaderView.swift

```diff
@@ -0,0 +1,65 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+ import Foundation
+ import Common
+
+final class PasswordGeneratorHeaderView: UIView, ThemeApplicable {
+    private enum UX {
+        static let headerIconLabelSpacing: CGFloat = 10
+        static let headerVerticalPadding: CGFloat = 8
+        static let headerImageHeight: CGFloat = 24
+    }
+
+    private let scaledHeaderImageSize = UIFontMetrics.default.scaledValue(for: UX.headerImageHeight)
+
+    private lazy var headerLabel: UILabel = .build { label in
+        label.numberOfLines = 0
+        label.font = FXFontStyles.Bold.body.scaledFont()
+        label.text = .PasswordGenerator.Title
+        label.isAccessibilityElement = true
```

Comment:
Nit: You should not need to enable a UILabel as an a11y element - they are by default.